lms/djangoapps/ccx/tests/utils.py

```diff
@@ -77,6 +85,17 @@ def setUp(self):
 
         # Create instructor account
         self.coach = AdminFactory.create()
+
+        # adding staff to master course.
+        self.staff = UserFactory()
```

Comment:
Instead of creating the staff here, can you move `make_staff` and `make_instructor` from `TestAdminAccessCoachDashboard` up to this class `CcxTestCase`? Then you should use those functions in the tests you wrote to create the staff and instructor.